src/updateIn.ts

```diff
@@ -1,30 +1,37 @@
-// @flow
-
 import { getIterator } from 'iterall'
 
-function _updateIn<T>(obj: T, path: Iterable<any>): T {
-  const notSetValue = arguments.length === 4 ? arguments[2] : undefined
-  const updater = arguments.length === 4 ? arguments[3] : arguments[2]
-
+function updateIn<T>(
+  obj: T,
+  path: Iterable<any>,
```

Comment:
@forrest-keller it would be best to add an overload for calling with `(obj, path, updater)` since `notSetValue` is an optional param but not at the end

Reply:
pushed a change